pkg/apis/admission/install/roundtrip_test.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/testing/roundtrip"
+	admissionfuzzer "k8s.io/kubernetes/pkg/apis/admission/fuzzer"
+)
+
+func TestRoundTrip(t *testing.T) {
+	roundtrip.RoundTripTestForAPIGroup(t, Install, admissionfuzzer.Funcs)
```

Comment:
@caesarxuchao well, the centralized roundtrip test is enough to capture all the api types imported from `k8s.io/kubernetes/pkg/api/testap`.

I think the problem is whether we are consistent everywhere, see:

```console
$ find staging/* -name roundtrip_test.go
staging/src/k8s.io/apiextensions-apiserver/examples/client-go/apis/cr/v1/roundtrip_test.go
staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install/roundtrip_test.go
staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/roundtrip_test.go
staging/src/k8s.io/apimachinery/pkg/apis/testapigroup/install/roundtrip_test.go
staging/src/k8s.io/apiserver/pkg/apis/example/install/roundtrip_test.go
staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/install/roundtrip_test.go
```